vaadin-crud-flow/src/main/java/com/vaadin/flow/component/crud/CrudEditorPosition.java

```diff
@@ -0,0 +1,59 @@
+package com.vaadin.flow.component.crud;
+
+/*
+ * #%L
+ * Vaadin Crud
+ * %%
+ * Copyright (C) 2018 Vaadin Ltd
+ * %%
+ * This program is available under Commercial Vaadin Add-On License 3.0
+ * (CVALv3).
+ * 
+ * See the file license.html distributed with this software for more
+ * information about licensing.
+ * 
+ * You should have received a copy of the CVALv3 along with this program.
+ * If not, see <http://vaadin.com/license/cval-3>.
+ * #L%
+ */
+
+import java.util.Arrays;
+
+/**
+ * Enum with the possible values for the crud's editor position.
+ */
+public enum CrudEditorPosition {
+
+    /**
+     * Editor form is positioned inside the overlay
+     */
+    OVERLAY(""),
+
+    /**
+     * Editor form is positioned below the grid
+     */
+    BOTTOM("bottom"),
+
+    /**
+     * Editor form is positioned on the grid side
+     * `right` - if lft
+     * `left` - if rtl
```

Comment:
The format of this javadoc is not looking nice on IDEA (look below):
![image](https://user-images.githubusercontent.com/262432/67003479-b6b57380-f0e6-11e9-9a61-d24b3bb0f16b.png)

VS Code is looking kind of the same as well.

One suggestion would be to change it to something like this (though VS Code is not respecting `<br>` tag):

```diff
-     * `right` - if lft
-     * `left` - if rtl
+     * <p>
+     * - <code>right</code> - if lft <br>
+     * - <code>left</code> - if rtl
```

Reply:
Done.